source/features/confirm-too-short-comments.tsx

```diff
@@ -0,0 +1,47 @@
+// import select from 'select-dom';
+import delegate from 'delegate-it';
+import * as pageDetect from 'github-url-detection';
+
+import features from '.';
+
+
+// function handleComment() {
+function handleComment(event: delegate.Event<MouseEvent, KeyboardEvent, HTMLAnchorElement>): void {
+    let body = document.querySelector('textarea[name="comment[body]"]').value.length;
+    if (body < 3 && !confirm('Body too short, sure?')) {
```

Comment:
The text wasn’t to be taken literally 😃

Reply:
Yes, of course, but it seemed short and straightforward, so I thought to keep it in this initial stage.
I'll rephrase it.

Reply:
I changed it into `Less than 3 chrs, are you sure?`.

Reply:
That’s not grammatically correct and has a typo

Reply:
Ok, I've changed the questions to `Body/Title is less than 3 characters, are you sure?`
I also made separate questions for body and title.